netket/models/slater.py

```diff
@@ -63,29 +63,29 @@ class Slater2nd(nn.Module):
     Assume we introduce a set of orbitals :math:`\phi_\mu(r, s)` with orbital index :math:`\mu`.
 
     - **Generalized Hartree-Fock** (:code:`generalized = True`) is the most general case
-      where we impose no restrictions. In particular, we do not restrict the orbitals
-      to have definite spin or orbital quantum numbers.
-      The total number of parameters is :math:`n_{\mathrm{M}} \times n_{\mathrm{f}}`. Hence,
-      any fermion can occupy any of the fermionic modes.
+    where we impose no restrictions. In particular, we do not restrict the orbitals
+    to have definite spin or orbital quantum numbers.
+    The total number of parameters is :math:`n_{\mathrm{M}} \times n_{\mathrm{f}}`. Hence,
+    any fermion can occupy any of the fermionic modes.
     - **Hartree-Fock (Spin-Conserving)** (:code:`[generalized=False,] restricted=True/False`).
-      Most physical Hamiltonians are spin conserving, and hence we can impose it also on the
-      wave-function. In this case, we separate the orbital index :math:`\mu \to (l, \alpha)` into
-      a spin and spatial orbital part: :math:`\phi_\mu(r, s)=\varphi_{l,\alpha}(r) \chi_{\alpha}(s)`.
-      Here, :math:`l` and :math:`\alpha` indicate the orbital and spin quantum numbers associated
-      with the orbital, and :math:`(r, s)` are the position vector and spin quantum number at which
-      we aim to evaluate the orbital (i.e. properties of a given fermion). Furthermore,
-      :math:`\varphi_{l,\alpha}(r)` is the spatial orbital at position :math:`r`, and and
-      :math:`\chi_\alpha(s)` the spin part.
-
-      - **Unrestricted Hartree Fock (UHF)** (:code:`[generalized=False,] restricted=False`),
```

Comment:
@jwnys You make Sphinx complain.

The reason those are indented is because sphinx wants them indented in this particular way.

I guess that you are annoyed by vscode flagging it as red, but this breaks the documentation so I'm going to revert it.

I'd love to know how to tell vscode not to complain.

Reply:
Hm, I didn't change those myself, nor did I allow vscode to change anything, this is either ruff or one of the pre-commit methods, is that possible?

Reply:
I doubt that because they do not do those kind of changes. 

I would put my money on auto format in vs code.

anyhow. Not important.